site/index.ejs

```diff
@@ -36,6 +36,74 @@
         }
     </script>
     <title><% if (typeof title !== 'undefined') {%><%= title %><% } %></title>
+
+    <script>
+    var colourOnButtonText = 'Make my day accessible';
```

Comment:
Looks like the indentation is a bit off here

Reply:
thanks, updated!